next-i18next.config.js

```diff
@@ -1,7 +1,7 @@
 /** @type {import('next-i18next').UserConfig} */
 module.exports = {
-    i18n: {
-      defaultLocale: 'en',
-      locales: ['en', 'ar'],
-    },
-  }
\ No newline at end of file
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en', 'ar'],
+  },
+};
```

Comment:
the commit message says called `add i18next configs`
but it didn't add them, it just fixed some indentations in here, and also this commit is deleting some files, so the commit message is misleading, pay more attention to the commit messages